Sources/LambdaKit/UIWebView+LambdaKit.swift

```diff
@@ -25,7 +25,7 @@
 import Foundation
 import UIKit
 
-public typealias LKShouldStartClosure = (UIWebView, URLRequest, UIWebViewNavigationType) -> Bool
+public typealias LKShouldStartClosure = (UIWebView, URLRequest, UIWebView.NavigationType) -> Bool
```

Comment:
Since `UIWebView` is now formally deprecated we should probably just remove this file altogether

Reply:
We need to change our internal stuff for this first